src/pages/programs.js

```diff
@@ -0,0 +1,61 @@
+import { graphql} from 'gatsby';
+import Layout from '../components/layout';
+import React from 'react';
+import Seo from '../components/seo';
+import '../styles/programsList.css';
+
+const ProgramsPage = ({ data }) => (
+    <Layout>
+      {data.allNodeProgram.edges.sort(function(a,b){
+        var titleA =a.node.title, titleB = b.node.title;
+        if (titleA < titleB)
+          return -1;
+        if (titleA > titleB)
+          return 1;
+        return 0;
+      })}
+        <Seo title="Program List" keywords={[`gatsby`, `application`, `react`]} />
```

Comment:
```suggestion
        <Seo title="Program List" />
```

The `keywords` field is no longer recognized by reputable search engines. We could replace it with `description="whatever"` but someone else (either from Admissions or C&PA) would need to decide how this Program Listing page should be described.

Reply:
will bring it up at the huddle tomorrow